backend/src/Notifo.Domain.Integrations.Abstractions/IntegrationProperty.cs

```diff
@@ -185,8 +185,10 @@ private bool TryGetString(string? input, [MaybeNullWhen(true)] out string error,
 
                     break;
                 case PropertyFormat.HttpUrl:
-                    // We only allow http and https to enable the usage of URL field for HttpClient requests.
-                    if (!Uri.TryCreate(input, UriKind.Absolute, out var uri) || !((string[])["http", "https"]).Contains(uri.Scheme, StringComparer.OrdinalIgnoreCase))
+                    // We only allow "http" and "https" schemas to enable the usage of URL field for HttpClient requests.
+                    if (!Uri.TryCreate(input, UriKind.Absolute, out var uri)
+                        || (!string.Equals(uri.Scheme, "http", StringComparison.OrdinalIgnoreCase) && !string.Equals(uri.Scheme, "https", StringComparison.OrdinalIgnoreCase))
```

Comment:
You can just create a static readonly field with HttpSchemes, would have been shorter ;)

Reply:
Should I do it in another pull request? I see you already merged this one :)

Reply:
Yes, plz :)